roerich/algorithms/calc_metrics.py

```diff
@@ -0,0 +1,38 @@
+from abc import ABCMeta, abstractmethod
+import numpy as np
+import pandas as pd
+from sklearn.metrics import pairwise_distances, roc_curve, roc_auc_score
+from sklearn.model_selection import StratifiedKFold, KFold, train_test_split
+from sklearn.discriminant_analysis import QuadraticDiscriminantAnalysis
+from copy import deepcopy
+from joblib import Parallel, delayed
+from scipy import interpolate
+from scipy.signal import argrelmax
+
+from .cpdc import ChangePointDetectionBase
+from roerich.scores.fd import frechet_distance
+from roerich.scores.mmd import maximum_mean_discrepancy
+
+
+class ScoreCalculator(ChangePointDetectionBase):
+
+    def __init__(self, metric=None, func=None, periods=1, window_size=100, step=1, n_runs=1):
```

Comment:
Add docstring with description of all input hyperparameters, please. For an example, https://github.com/HSE-LAMBDA/roerich/blob/main/roerich/algorithms/cpdc_cv.py